shifthelper/conditions.py

```diff
@@ -242,6 +242,7 @@ def is_no_dimctrl_server_available():
 
 @log_call_and_result
 def is_no_shift_at_the_moment():
+    '''no shift at the moment'''
```

Comment:
this is an important bugfix, which has nothing to do with logging. 
So it is good to have it, but would have been good to at least mention it in the discussion of this PR, I guess.